src/Libraries/CoreNodeModels/Input/SliderBase.cs

```diff
@@ -130,5 +131,27 @@ internal static int ConvertStringToInt(string value)
             }
             return result;
         }
+
+        internal static long ConvertStringToInt64(string value)
```

Comment:
Can you add some function comments saying what would be the expected return value for string over the maximum or minimum value?

Reply:
Sure.